client/src/components/App.jsx

```diff
@@ -1,33 +1,60 @@
 import React from 'react';
-
+import $ from 'jquery';
+import Header from './Header.jsx';
+import Menu from './Menu.jsx';
+import Sidebar from './Sidebar.jsx';
+import exampleData from './exampleData.js';
 
 class App extends React.Component {
-    constructor(props){
+    constructor(props) {
         super(props);
         this.state = ({
```

Comment:
You might have extra parenthesis around your state?

Reply:
great catch!